src/components/Controls/Controls.js

```diff
@@ -0,0 +1,20 @@
+import './Controls.css';
+
+export default function Controls({ setContinent, countries }) {
+  //   const countries = useCountries();
```

Comment:
good job commenting this out — as Julie pointed out in lecture you only want to have one instance of your ```useWhatever()``` hook and then you'll pass the values from that down as props